tbx/work/models.py

```diff
@@ -43,9 +33,11 @@ class WorkPageScreenshot(Orderable):
         FieldPanel("image"),
     ]
 
+    class Meta:
+        abstract = True
+
 
-class WorkPageAuthor(Orderable):
-    page = ParentalKey("work.WorkPage", related_name="authors")
+class AbstractPageAuthor(models.Model):
```

Comment:
Let's avoid creating duplicate versions of these models for each page type - it's unnecessary. Just ALWAYS use `wagtailcore.Page` in the `ParentalKey` - Which options appear in the Wagtail admin UI for each page type is controlled via panels.

Reply:
Thanks @ababic I've added a couple of commits to remove these extraneous models. 
I've also added a data migration to ensure that we don't lose any author info for existing pages.